### ISSUE TITLE
Logical structure of HTML - lists

### ISSUE THREAD

**Issue by Angamanga** [ushahidi/platform#2749]
**Go through pattern-library and check all HTML-lists we are using with the following in mind:**
- Unordered lists should be used when there is no order of sequence or importance. 
- Ordered lists suggest a progression or sequence. 
- Definition lists should be used explicitly for presenting a structure for definitions.
- Unordered and ordered lists should always contain list items.
- Definition lists must always have definition descriptions. 
- Empty lists are incorrect HTML. Lists should never be used for merely indenting or other layout purposes. Nested lists should be coded properly.

- fix where neccessary

**Comment by rowasc** [comment]
@Angamanga @sethburtonhall is this in the correct pipeline ? (Ready for Dev)
